.github/workflows/conda-build.yml

```diff
@@ -7,6 +7,8 @@ on:
     branches:
       - master
 jobs:
+  call-ci-image-update-job:
+    uses: ./.github/workflows/ubuntu-ci-image-update.yml
```

Comment:
supurfluous, this gets called as part of call-ubuntu-build-job

Reply:
removed